include/core/CMemoryDec.h

```diff
@@ -80,25 +80,27 @@ std::size_t bucketGroupOverhead(const T&) {
 }
 
 //! The overhead associated with the unordered container's \em bucket_groups.
-//! This is calculated as 4 bits per bucket_group.
+//! This is calculated as 4 bits per bucket_group pointer, of which there are 4.
```

Comment:
```suggestion
//! On average this is 4 bits per bucket. This is because the bucket_group
//! structure consists of a 64 bit bitmask representing up to 64 buckets,
//! plus 3 pointers, so a 32 byte structure stores the information for up to
//! 64 buckets.
```

And the same comment change can be made below too.